### PR TITLE
wireguard and iptables

### PR DESCRIPTION
**Pull request by neo-liang-sap**
**What this PR does / why we need it**:
wireguard and iptables
**Which issue(s) this PR fixes**:
Fixes https://github.com/gardener/ops-toolbelt/issues/46

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

File: dockerfile-configs/common-components.yaml [hunk 1]
```diff
@@ -74,6 +76,12 @@
       apt-get --yes update && apt-get install --yes locales ;\
       locale-gen "en_US.UTF-8"
     info: ~
+  - name: iptables-persistent
+    command: |
+      echo iptables-persistent iptables-persistent/autosave_v4 boolean true | debconf-set-selections ;\
+      echo iptables-persistent iptables-persistent/autosave_v6 boolean true | debconf-set-selections ;\
+      sudo apt-get ---yes install iptables-persistent
```
Comment (plkokanov) [inline dockerfile-configs/common-components.yaml:83]:
```suggestion
      sudo apt-get --yes install iptables-persistent
```

**Review by plkokanov — CHANGES_REQUESTED** [review]
Thanks for the PR!
Can you check how much the size of the image increases after adding these components?

File: dockerfile-configs/common-components.yaml (hunk 1 above)
Comment (neo-liang-sap) [reply dockerfile-configs/common-components.yaml:83]: thanks for the catch and sorry for typo

**Comment by neo-liang-sap** [comment]
Hi @plkokanov , the image increased ~15mb from my local verification
image

**Comment by neo-liang-sap** [comment]
close per discussion with @plkokanov

**Pull request closed without merging**